kolena/_experimental/dataset/_dataset.py

```diff
@@ -251,9 +253,12 @@ def register_dataset(
                 validate_dataframe_ids(chunk, id_fields)
                 validated = True
             _upload_dataset_chunk(chunk, load_uuid, id_fields)
+
     request = RegisterRequest(name=name, id_fields=id_fields, uuid=load_uuid)
     response = krequests.post(Path.REGISTER, json=asdict(request))
     krequests.raise_for_status(response)
+    data = from_dict(EntityData, response.json())
+    log.info(f"Successfully created/updated dataset {name} {get_dataset_url(dataset_id=data.id)}")
```

Comment:
In existing messages, we're careful to always surround names with single quotes (`'{name}'`) and URLs in parentheses (`({url})`)

Reply:
Also, can we think of a better verb than `created/updated`?

Reply:
Updated to use `registered` as it fits both creating and updating the dataset. Also added the single quotes and parentheses